app/views/service_toolkit/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<% add_view_stylesheet("service-toolkit") %>
+<%
+  content_for :title, "Service Toolkit - GOV.UK"
```

Comment:
Nitpick: I think you could combine this section with the `add_view_stylesheet` section above.

Reply:
@andysellick Thanks, have made this change.